ast/format_test.go

```diff
@@ -0,0 +1,222 @@
+package ast_test
+
+import (
+	"testing"
+
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/parser"
+	"github.com/pkg/errors"
+)
+
+var skip = map[string]string{
+	"array_expr":           "without pars -> bad syntax, with pars formatting removes them",
+	"conditional":          "how is a conditional expression defined in spec?",
+	"duration_multiple":    "how are multiple duration values meant to be used?",
```

Comment:
This test case seems valid. What is the question? Durations should be formatted as largest to smallest duration units.

Reply:
Now I got it, forget about it 😄